example/app/data/DemoDataSource/apps/MySignalApp/models/CaseProxy.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "CaseProxy",
+  "type": "CORE:Blueprint",
+  "extends": ["CORE:NamedEntity"],
+  "description": "",
+  "attributes": [
+    {
+      "name": "type",
+      "attributeType": "string",
+      "type": "CORE:BlueprintAttribute"
```

Comment:
is type needed in attributes list if we extend from NamedEntity?

Reply:
No. Removed